pandas/tests/frame/test_indexing.py

```diff
@@ -208,6 +208,29 @@ def test_setitem_list_of_tuples(self, float_frame):
         expected = Series(tuples, index=float_frame.index, name="tuples")
         assert_series_equal(result, expected)
 
+    @pytest.mark.parametrize("columns", [["A", "E"], ["E", "F"]])
+    @pytest.mark.parametrize(
+        "box",
+        [
+            lambda x: 1,
+            lambda x: [1, 2],
+            lambda x: np.array([1, 2]),
+            lambda x: x[["B", "C"]],
+            lambda x: x[["B", "A"]].values,
+            lambda x: x[["A", "C"]].values.tolist(),
+        ],
+    )
+    def test_setitem_list_missing_columns(self, float_frame, columns, box):
+        # GH 26534
+        result = float_frame.copy()
+        result[columns] = box(float_frame)
```

Comment:
Hmm not really sure what the point of this parametrization is - can you construct test(s) that don't overwrite the existing column data before the assignment? Seems unrelated unless I am missing something

Reply:
I parametrize the tests as https://github.com/pandas-dev/pandas/pull/26534#discussion_r306287424 suggest. What do you propose?